book/src/user/docker.md

```diff
@@ -26,8 +26,96 @@ docker run --detach zebra:local
 
 See [Building Zebra](https://github.com/ZcashFoundation/zebra#building-zebra) for more information.
 
+## Advanced usage
+
+You're able to specify various parameters when building or launching the Docker image, which are meant to be used by developers and CI pipelines. For example, specifying the Network where Zebra will run (Mainnet, Testnet, etc), or enabling features like mining.
+
+For example, if we'd like to enable mining on the image, we'd build it using the following `build-arg`:
```

Comment:
Any other good example that we can use and it might be something non-dev users would use?

Reply:
Let me know of any other example you'd like to use, or if we can continue with this one and improve later on.

Reply:
A good one is `prometheus`: https://zebra.zfnd.org/user/metrics.html, but that requires running Prometheus and Grafana, as described in the link. We could just mention 

Another one is `progress-bar`. That one doesn't require anything extra, but I don't know if users will be interested in that since they will run Zebra in Docker. We might also remove the feature if we decide we want the progress bars by default.

Also, the `getblocktemplate-rpcs` feature might be removed once mining is not experimental.

I think the best way forward would be to add docs for running Zebra in Docker here: https://zebra.zfnd.org/user/metrics.html. We would then have a nice example feature for Docker, which would be `prometheus`.

Reply:
The docs for Prometheus already describe how to run it together with Grafana in separate Docker containers. Having a third container with Zebra would be a nice fit.

Reply:
When looking for examples, I noticed a few bugs in the docs. I fixed them in #7567.

Reply:
This should be good now. When we have at least a `docker-compose`, then we can make a full example with it (as it's way better than handling individual containers)

Reply:
Sounds good.